app/serializers/user.js

```diff
@@ -5,9 +5,13 @@ export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (snapshot.id) {
-      let attributesToOmit = ['email'];
+      let attributesToOmit = [];
       if (!snapshot.adapterOptions || !snapshot.adapterOptions.includePassword) {
         attributesToOmit.push('password');
+      } else if (!snapshot.adapterOptions || !snapshot.adapterOptions.includeEmail) {
```

Comment:
@niranjan94 But the thing is api is not accepting. It is demanding the email and password field.
![email-api4](https://user-images.githubusercontent.com/17925669/28497835-94376294-6fad-11e7-9ed6-8f5def7a4f5f.png)